modules/core/src/main/scala/dk/cwconsult/peregrin/core/Migrations.scala

```diff
@@ -8,8 +8,10 @@ object Migrations {
 
   /**
     * Apply a list of migrations to the given database schema.
+    *
+    * Must all be of the same Migration definition format.
     */
-  def applyMigrations(connection: Connection, schema: Schema, migrations: Seq[Migration]): AppliedMigrations = {
+  def applyMigrations[M <: Migration](connection: Connection, schema: Schema, migrations: Seq[M]): AppliedMigrations = {
```

Comment:
I think I'll make this one private, and expose overloaded versions for each of the Migration-subtypes, so we ensure don't get mixed migration versions in the sequence.